csharp/Platform.Collections/Arrays/CharArrayExtensions.cs

```diff
@@ -77,11 +77,11 @@ private static bool CheckArraysMainPartForEquality(ref char* left, ref char* rig
         {
             while (length >= 10)
             {
-                if ((*(int*)left != *(int*)right)
```

Comment:
Why.....?

Reply:
What `why`?

Reply:
@uselessgoddess, why did you removed braces? They make code less error prone

Reply:
Sorry dear @FreePhoenix888, my ide did this

Reply:
But braces make code error prone:)

Reply:
I disagree

Reply:
Okay: https://github.com/linksplatform/Collections/pull/148/files#r945110665